src/UdtException.cs

```diff
@@ -24,13 +25,13 @@ public UdtException(int major = 0, int minor = 0, int err = -1)
                 m_iErrno = err;
         }
 
-        public string getErrorMessage()
+        private static string getErrorMessage(int major, int minor)

```

Comment:
I actually replaced all this code already in our internal repo @gwynroberts haven't you published the latest changes??